pytorch_lightning/loops/batch/training_batch_loop.py

```diff
@@ -421,13 +441,17 @@ def _optimizer_step(
             using_lbfgs=is_lbfgs,
         )
 
```

Comment:
Could this be the place where we increment the progress? This would be the same, right? Or wouldn't this work for manual optimization?

Reply:
Yes, moved it there, and yes it won't work for manual optimization.

Reply:
We will handle manual optimization in another PR.